partials/_floating.scss

```diff
@@ -0,0 +1,142 @@
+.floating {
+    margin-top: 52px;
+    width: 100%;
+    display: flexbox;
+    flex-direction: column;
+    position: relative;
+    margin: auto;
+    
+
+    @include media ( ">=md" ) {
+        text-align: center;
+        padding: 0;
+        max-width: 768px;
+    }
+
+    @include media ( ">xl" ) {
+        max-width: 1024px;
+    }
+
+    &__block {
+        flex-direction: row;
```

Comment:
this won't have any effect if you haven't added `display: flex;` to this element

Reply:
Have just removed this as it works the correct way without, is that ok?

Reply:
Yes :)